docs/reference/indices/segments.asciidoc

```diff
@@ -1,10 +1,10 @@
 [[indices-segments]]
 == Indices Segments
 
-Provide low level segments information that a Lucene index (shard level)
-is built with. Allows to be used to provide more information on the
-state of a shard and an index, possibly optimization information, data
-"wasted" on deletes, and so on.
+Provides low level segments information that a Lucene index 
```

Comment:
I like this change in general, but maybe you could find a way to keep the information that a Lucene index is equivalent to an Elasticsearch shard. I think that's what was ment by the bracketed part.